enterprise/cypher/slotted-runtime/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/slotted/SlottedQueryState.scala

```diff
@@ -72,10 +72,42 @@ case class SlottedExecutionContextFactory(slots: SlotConfiguration) extends Exec
     newCtx
   }
 
+  override def copyWith(row: ExecutionContext, newEntries: Seq[(String, AnyValue)]): ExecutionContext = {
+    val newCopy = SlottedExecutionContext(slots)
+    row.copyTo(newCopy)
+    for (t <- newEntries) {
```

Comment:
```
for ((key,value) <- newEntries) {...
```

Reply:
Thanks, I was somehow under the impression that scalac didn't support that for some ambiguity reason. Maybe from some previous scala version? Much better now.